source/refined-github.css

```diff
@@ -1,8 +1,8 @@
 :root {
-	--rgh-green: var(--color-success-fg, #1a7f37);
-	--rgh-red: var(--color-danger-fg, #cf222e);
-	--rgh-border-color: var(--color-border-muted, #d8dee4);
-	--rgh-background: var(--color-canvas-default, #fff);
+	--rgh-green: var(--fgColor-success, #1a7f37);
```

Comment:
Can we fallback to the old for GHE?

Reply:
Ah, good point, I was wondering whether GHE would need backcompat on it.
Would that be var(GH, var(GHE, default))?

Reply:
From what I'm reading, it looks like var(var1, fallback1, fallback2) behavior is inconsistent across browsers, so I'm going with nested var.

Reply:
Yea, I believe you have to do `var(GH, var(GHE, default)`